gramps/gen/filters/rules/family/_hasnote.py

```diff
@@ -46,3 +46,4 @@ class HasNote(HasNoteBase):
 
     name = _("Families having <count> notes")
     description = _("Matches families having a certain number notes")
+    table = "family"
```

Comment:
I could not work out what this was for? 
The filter appears to work with this line commented out.